response_operations_ui/controllers/collection_exercise_controllers.py

```diff
@@ -161,6 +161,26 @@ def get_case_group_status_by_collection_exercise(case_groups, collection_exercis
                 if case_group['collectionExerciseId'] == collection_exercise_id)
 
 
+def unlink_sample_summary(collection_exercise_id, sample_summary_id):
+    logger.debug('un-linking sample summary from collection exercise', collection_exercise_id=collection_exercise_id,
+                 sample_summary_id=sample_summary_id)
+    url = f'{app.config["COLLECTION_EXERCISE_URL"]}/collectionexercises/unlink/{collection_exercise_id}' \
+          f'/sample/{sample_summary_id}'
+
+    response = requests.put(url, auth=app.config['COLLECTION_EXERCISE_AUTH'])
```

Comment:
The put call has now changed to delete `requests.delete(url, auth=app.config['COLLECTION_EXERCISE_AUTH'])`